src/test/java/net/minestom/server/inventory/InventoryTest.java

```diff
@@ -78,9 +78,7 @@ public void testAdd() {
 
     @Test
     public void testIds() {
-        for (int i = 0; i <= 256; ++i) {
-            final Inventory inventory = new Inventory(InventoryType.CHEST_1_ROW, "title");
-            assertTrue(inventory.getWindowId() != 0);
-        }
+        for (int i = 0; i <= 1000; ++i)
+            assertTrue(new Inventory(InventoryType.CHEST_1_ROW, "title").getWindowId() > 0);
```

Comment:
Check if the value is within [1, 127] since this is the valid range

Reply:
The window id method's return value is a byte so I don't believe we should change this current check.

Reply:
alright